controls/raddiagram/diagram-items/features-container-shapes.md

```diff
@@ -242,36 +242,81 @@ Below you can find a list of all __RadDiagramContainerShape__ members that are r
 
 ## Customize the ContainerShape Appearance
 
 You can easily customize the visual appearance of the __RadDiagramContainerShape__ by using the following properties:		
 
-* __Background__ - gets or sets the brush that specifies the __RadDiagramContainerShape__ background color.			
+* __Background__: Gets or sets the brush that specifies the __RadDiagramContainerShape__ background color.			
 
-* __BorderBrush__ - gets or sets the brush that specifies the __RadDiagramContainerShape__ border color.			
+* __BorderBrush__: Gets or sets the brush that specifies the __RadDiagramContainerShape__ border color.			
 
-* __BorderThickness__ - gets or sets the width of the __RadDiagramContainerShape__ outline.			
+* __BorderThickness__: Gets or sets the width of the __RadDiagramContainerShape__ outline.			
 
 >tip You can use the __RadDiagram.ContainerShapeStyle__ property to explicitly apply a style on all __RadDiagramContainerShapes__ in a __RadDiagram__ instance.		  
 
+## Container Service
+
+* __CompleteManipulation__: A void method that completes given manipulation. Accept parameter of type __CompositeAsyncStateCommand__.
+* __DetachItemsFromContainer__: A void method that detaches items from a container.
+* __GetChildrenBounds__: This method returns Rect element which wraps the items inside the container.
+* __GetContainerChildrenRecursively__: This method search recursively and return all children of the container.
+* __GetFloatingChildren__: This method return all shapes and connection which are not attched to shapes.
+* __GetParentContainersRecursively__: This method search recursively and return all parent containers.
+* __GetRootItems__: A void method the return all root containers if any exist. 
+* __UpdateContainerLayout__: A void method thats update the container layout. Accept parameter of type __IDragDropAware__.
+
+## How To
+
+* __Prevent dropping a shape in container__: To prevent dropping a shape inside a container, you can set the __IsDropEnabled__ property of the __RadDiagramContainerShape__ to __False__.
+
+## IDragDropAware interface
+
+__RadDiagramContainerShape__ implements __IDragDropAware__ interface. This interface allows you to get notify when a shape is dragged over a container. The interface implements the following events and property:
+
+* __IsDropPossible__: A property of type __bool__ that gets whether drag-drop items in this container is possible.
+* __OnDragEnter__: This event is called when a dragged shape is over a container.
+* __OnDragLeave__: This event is called when a dragged shape leave the boundary of the currently hover container.
+* __OnDrop__: This event is called when a dragged shape is drop over a container.
+
+To subsribe to these events you can create a custom class which derives from __RadDiagramContainerShape__ and override these methods.
+
+#### __[C#] Example : Overriding drag/drop events of container shape__

```

Comment:
There is no overriding of events. Instead you can say overriding event handlers. Or overriding drag/drop methods.